### ISSUE TITLE
Learn mode: No data received

### ISSUE THREAD

**Issue by Komagain** [mjg59/python-broadlink#186]
Hello
Each time I try to make my RM learn (IR or RF), the following message is displayed:
```
Learning
No data received
```
I am able to learn via ihc app
What can I check ?

Thanks

**Comment by Nightreaver** [comment]
What are you trying to learn? IR ?

**Comment by Komagain** [comment]
I have tried both IR and RF
It works with the broadlink app.

**Comment by rwa** [comment]
I am having the same problem, only with RF devices.  I can learn them on the ihc app, but I always time out trying to learn them through the cli.  (I can learn IR codes just fine).

My device type is being reported as 0x279d, which seems to be new.  Is it possible the protocol has been changed for this device?

**Comment by vchoi** [comment]
I'm having the same issue while trying to learn IR codes. Device 0x2787

<a href="https://asciinema.org/a/0CUdIOfe5kEMz8SAX9BKRrv6J" target="_blank"><img src="https://asciinema.org/a/0CUdIOfe5kEMz8SAX9BKRrv6J.svg" /></a>

**Comment by mtronicsDev** [comment]
> I am having the same problem, only with RF devices. I can learn them on the ihc app, but I always time out trying to learn them through the cli. (I can learn IR codes just fine).
> 
> My device type is being reported as 0x279d, which seems to be new. Is it possible the protocol has been changed for this device?

I have the same device type and have been able to learn RF exactly once. Everytime apart from the one where it worked, I always got past the RF sweeping part but then, no code was found.

The one time it worked, I got a code which I could send without a problem.

**Comment by zero1niner** [comment]
Having the same problem

**Comment by zero1niner** [comment]
wait..I found a solution in my situation. Hoping my hours of pain will help someone else. 

Someone gave me this tip, which opens the help doc
./broadlink_cli -h

From there, I saw the --rfscanlearn command. 

Changed my string to have the --rfscanlearn at the end instead of the learn i had been using, and suddenly the thing started learning my remotes. 

I have spent DAYS on this, and if I were only a better developer, I would commit to writing up a full how to that is complete and current. But im not clear enough on the steps, and I would only confuse people more by trying. 

For RF devices, I changed the 

./broadlink_cli --type 0x649b --host 192.168.1.142 --mac a043b018f07b --rfscanlearn
